src/databricks/labs/ucx/upgrades/v0.16.0_changing_cluster_table_schema.py

```diff
@@ -0,0 +1,25 @@
+# pylint: disable=invalid-name
+
+import logging
+from typing import Any
+
+from databricks.labs.blueprint.installation import Installation
+from databricks.sdk import WorkspaceClient
+
+from databricks.labs.ucx.assessment.clusters import ClusterInfo
+from databricks.labs.ucx.config import WorkspaceConfig
+from databricks.labs.ucx.framework.crawlers import (
+    SchemaDeployer,
+    StatementExecutionBackend,
+)
+
+logger = logging.getLogger(__name__)
+
+
+def upgrade(installation: Installation, ws: WorkspaceClient):
+    config = installation.load(WorkspaceConfig)
+    sql = f"DROP TABLE IF EXISTS hive_metastore.{config.inventory_database}.clusters"
```

Comment:
Why are you dropping the production database table? 😰

Alter table and add columns instead

Reply:
I have changed the code to alter the table.
But why I used drop and recreate is that once we drop and recreate and then if customer run the assessment it should populate the correct data.